pkgs/applications/audio/picard/default.nix

```
@@ -1,9 +1,11 @@
-{ lib
+{ fetchFromGitHub
+, lib
 , python3Packages
```

```suggestion
{ lib
, python3Packages
, fetchFromGitHub
```

See the last line of https://nixos.org/manual/nixpkgs/stable/#sec-syntax.